shell/platform/darwin/ios/platform_view_ios.mm

```diff
@@ -132,6 +120,37 @@ new PlatformMessageHandlerIos(task_runners.GetPlatformTaskRunner())) {}
   };
 }
 
+void PlatformViewIOS::handleAccessibilityMessage(__weak id message, FlutterReply reply) {
+  if (!owner_controller_) {
+    FML_LOG(WARNING) << "Could not accept accessibility message, this "
+                        "PlatformViewIOS has no ViewController.";
+  }
+  NSString* type = message[@"type"];
+  if ([type isEqualToString:@"generatingSemanticsTree"]) {
+    BOOL generating = [message[@"data"][@"generating"] boolValue];
+    if (generating) {
+      if (!accessibility_bridge_) {
+        accessibility_bridge_ = std::make_unique<AccessibilityBridge>(owner_controller_, this,
+                                                                      platform_views_controller_);
```

Comment:
Can I assume `platform_views_controller_` contains the same `owner_controller_.platformViewsController`. From the code I think yes, but not too sure.